web/src/graphQL.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { useQuery, gql } from "@apollo/client";
+
+const GET_PROJECTS = gql`
+    query getProjects {
+        myProjects {
+            id
+        }
+    }
+`;
+export const GraphQL: React.FunctionComponent<{}> = () => {
+    const { loading, error, data } = useQuery(GET_PROJECTS);
+    if (loading) return <>"Loading..."</>;
+    if (error) return <>`Error! ${error.message}`</>;
+    console.log(data.myProjects);
+    return (
+        <>
+            {data.myProjects.map((item: { id: string }) => (
```

Comment:
Awesome! Do you get any type-safety here? Or would you basically have to create an interface for each endpoint?

Reply:
Added :)